source/_integrations/schedule.markdown

```diff
@@ -12,64 +12,52 @@ ha_domain: schedule
 ha_integration_type: helper
 ---
 
-The schedule integration provides a way to create a weekly schedule in
-Home Assistant that can be used to trigger or make decisions in your
-automations and scripts.
-
-The preferred way to configure a schedule is via the user interface at
-**{% my helpers title="Settings > Devices & services > Helpers." %}** Click the add button
-and then choose the **{% my config_flow_start domain=schedule title="Schedule" %}** option, or click the My button below.
+The **Schedule** {% term integration %} provides a way to create a weekly schedule {% term entity %} in Home Assistant that can be used to trigger or make decisions in your automations and scripts.
```

Comment:
We're going straight into time blocks a little bit more below, it would be great to explain in this introduction that a schedule consists of time blocks: 

```suggestion
The **Schedule** {% term integration %} provides a way to create a weekly schedule {% term entity %} in Home Assistant, consisting of time blocks with defined start and end times. The schedule activates when a time block starts and deactivates when it ends, allowing it to be used for triggering or making decisions in automations and scripts.
```